src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+$color-main: #060b35;
+$color-secondary: #616070;
+$buttom-main: #008bb0;
+$buttom-hover: white;
```

Comment:
There is a typo in the variable name `$buttom-hover`. It should be `$button-hover` to correctly represent the word 'button'.